packages/playground-examples/copy/ja/TypeScript/Primitives/Any.ts

```diff
@@ -0,0 +1,51 @@
+// AnyはTypeScriptのエスケープ句です。 
+// any型を利用することで、一区切りのコードを
+// JavaScriptの様に動的に扱ったり、
+// 型システムの制限を回避することができます。
+
+// any型を利用する良いケースはJSON parsingです
```

Comment:
I'm not confident here...

Reply:
IMO: 「JSONのparseです」が良さそう

Reply:
ここでのanyは推論されているものなので、「使用されている」のほうが自然かもしれません。
```suggestion
// any型が使用されている良い例はJSON.parseの結果です
```